### PR TITLE
Gas Meter Updates + QoL

### PR DESCRIPTION
**Pull request by DoctorSquishy**
# About The Pull Request
Fixes atmos pumps layering issue when pipes would connect to machines and would overlap the pumps
Adds Examine code that lets you see the total volume of a pipenet
Adds GAGS coloring to Gas Meters for temperatures.

    Grey (No pressure/No Temp/Unpowered/Broken)
    +700K Red
    +460K Orange
    +340K Yellow
    +270K Green (room temp)
    +200K Cyan
    +120K Blue
    +++1K Purple

## Why It's Good For The Game
You can now get a general idea of pipenet temperatures from across the room.

## Testing Photographs and Procedure
Tested various buildstates and pipe layers
<details>
<summary>Screenshots&Videos</summary>

image

</details>

## PR References

- https://github.com/tgstation/tgstation/pull/59924
- https://github.com/tgstation/tgstation/pull/60433

## Changelog
:cl: DoctorSquishy, timothymtorres
qol: Gas meters, now use GAGS coloring to display colors based on temperature. (red is hot, green is room temp, and purple is cold)
qol: Gas Meters also display a Pipenets Total Volume
imageadd: New Gas Meter Icons, they are now circular
fix: Fixes a strange layering overlap with pipe connections that would overlap any pump layer devices
/:cl:

### CODE REVIEW

File: code/datums/greyscale/greyscale_configs.dm [hunk 1]
```diff
@@ -126,3 +126,16 @@
...
 	name = "Space Carp Disk in Mouth"
 	json_config = 'code/datums/greyscale/json_configs/carp_disk_mouth.json'
+
+/datum/greyscale_config/janicart_upgrade
+	name = "Janicart Upgrade"
+	icon_file = 'monkestation/icons/obj/janitor/janicart_upgrade.dmi'
+	json_config = 'code/datums/greyscale/json_configs/janicart_upgrade.json'
+
+/datum/greyscale_config/janicart_upgrade/installed
+	json_config = 'code/datums/greyscale/json_configs/janicart_upgrade_installed.json'
```
Comment (nednaZ) [inline code/datums/greyscale/greyscale_configs.dm:136]: Unrelated change?

**Review by nednaZ — APPROVED** [review]
Looks fine to me, just make sure that the janicart move wasn't a mistake.

File: code/datums/greyscale/greyscale_configs.dm (hunk 1 above)
Comment (DoctorSquishy) [reply code/datums/greyscale/greyscale_configs.dm:136]: I was just moving them out of the modular folder since they were a complete icon replacement for the janicart attachments

Comment (DoctorSquishy) [reply code/datums/greyscale/greyscale_configs.dm:136]: No issues with the move

image
